src/pages/LikedFoodPage.js

```diff
@@ -3,27 +3,26 @@ import Card from '../components/Card.js';
 import Navigation from '../components/Navigation.js';
 
 export default function LikedFoodPage({ storageData }) {
-  const filteredLikedData = storageData?.filter(
-    data => data.foodJudge === 'liked'
-  );
+  const likedData = storageData?.filter(data => data.foodJudge === 'liked');
   return (
     <LikedFoodPageStyle>
       <Header>
         <h1>Mir schmeckt...</h1>
       </Header>
       <main>
-        {filteredLikedData.length === 0 ? (
+        {likedData.length === 0 ? (
           <p>
-            Du hast hier noch keine Listeneinträge. Listeneinträge erscheinen,
-            wenn das Formular ausgefüllt und gespeichert wird.
+            Du hast hier noch keine Listeneinträge. <br />
+            Listeneinträge erscheinen, wenn das Formular ausgefüllt und
+            gespeichert wird.
           </p>
         ) : (
           <ListStyle
             role="list"
             data-testid="likedlist"
             aria-describedby={'list of cards about catfood that your cat like'}
```

Comment:
I think for testing the role="list" would be enought...so the data-testid could be skipped!?

Reply:
yes, thanks i will change this 👍

Reply:
as there are more lists on this side according to conditional rendering testing didn't work as scheduled any more after removing and adjusting, so to have it clear for the test, which list should go in this test, i added the test-id again